marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx

```diff
@@ -200,6 +200,23 @@ describe('json scenario for snippet on browse documents page', () => {
     cy.contains('Show the processed data').should('be.visible');
   });
 
+  it('verify selected facet name persist after refreshing the page', () => {
+    browsePage.selectEntity('Person');
+    browsePage.getSelectedEntity().should('contain', 'Person');
+    browsePage.getFacetItemCheckbox('fname', 'Alexandra').click();
+    browsePage.getGreySelectedFacets('Alexandra').should('exist');
+    browsePage.getFacetApplyButton().click();
+    browsePage.getFacetItemCheckbox('fname', 'Alexandra').should('be.checked');
+    browsePage.getFacetSearchSelectionCount('fname').should('contain', '1');
+    cy.reload();
```

Comment:
`verify source view of the document` test is already testing for browser refresh. Can we pls add couple of steps there to verify this instead.

Reply:
Fixed. Thanks.